glide.yaml

```diff
@@ -0,0 +1,24 @@
+package: github.com/kubernetes-incubator/rktlet
+import:
+- package: k8s.io/kubernetes
```

Comment:
@s-urbaniak how do I add version restrictions actually? I want to restrict to the current HEAD's commit hash.